application/server/lsp/message_types.go

```diff
@@ -302,6 +302,7 @@ type Settings struct {
 	OsArch                      string         `json:"osArch,omitempty"`
 	RuntimeVersion              string         `json:"runtimeVersion,omitempty"`
 	RuntimeName                 string         `json:"runtimeName,omitempty"`
+	ScanningMode                string         `json:"scanningMode,omitempty"`
```

Comment:
why is this named differently than the config setting in `configuration`?

Reply:
This is a string, the config setting is a boolean specific to whether we do auto-scans or not.

It's easier to get a simple true/false about auto scans rather then comparing strings to "manual" everywhere.